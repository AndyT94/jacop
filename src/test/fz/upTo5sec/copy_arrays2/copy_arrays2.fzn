var 1..1: X_INTRODUCED_14 ::var_is_introduced ;
var 2..2: X_INTRODUCED_15 ::var_is_introduced ;
var 3..3: X_INTRODUCED_16 ::var_is_introduced ;
var 4..4: X_INTRODUCED_17 ::var_is_introduced ;
var 4..4: X_INTRODUCED_18 ::var_is_introduced ;
var 3..3: X_INTRODUCED_19 ::var_is_introduced ;
var 2..2: X_INTRODUCED_20 ::var_is_introduced ;
var 1..1: X_INTRODUCED_21 ::var_is_introduced ;
var 1..4: X_INTRODUCED_24;
array [1..4] of var int: x:: output_array([1..4]) = [1,2,X_INTRODUCED_24,4];
array [1..8] of var int: y:: output_array([1..4,1..2]) = [1,2,3,4,4,3,2,1];
solve  satisfy;
