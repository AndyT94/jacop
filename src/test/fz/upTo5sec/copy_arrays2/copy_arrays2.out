x = array1d(1..4, [1, 2, 1, 4]);
y = array2d(1..4, 1..2, [1, 2, 3, 4, 4, 3, 2, 1]);
----------